private/buf/bufgen/bufgenv1/config.go

```diff
@@ -75,9 +75,6 @@ func getConfig(
 		if err := unmarshalStrict(data, &externalConfigV1); err != nil {
 			return nil, err
 		}
-		if err := validateExternalConfigV1(externalConfigV1, id); err != nil {
```

Comment:
`validateExternalConfigV1` only validated plugin configs, now using `getPluginConfigs` to both validate and create plugin configs.